resources/security/security-for-beginners.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Here are some security considerations when developing apps for the Province of British Columbia
+resourceType: Documentation
+title: Considerations for App Security in the Public Sector
+tags:
+- Security
+- Starting Out
+- Beginner's Guide
+- STRA
+- PIA
+---
+
+# Considerations for App Security in the Public Sector
+
+This content is lonely and needs help! Some of the things we are looking to cover include:
+
+- What is an STRA
+- What is a PIA
+
```

Comment:
@jleach are there any other stubs you think we might want in this document? 

Some context for you is that i'm putting together a Topic in the Devhub for 'Developers who are new to the platform'